### PR TITLE
Vip 807 staging vit configurator

### PR DESCRIPTION
**Pull request by dwxander**
- creates identical VIT embed page pointing to staging embed
- set VIT environment (staging, prod) based on environment var
- adds new distribution to buildkite pipeline

### CODE REVIEW

File: src/clj/vit_configurator/env.clj [hunk 1]
```diff
@@ -0,0 +1,8 @@
+(ns vit-configurator.env
+  (:require [environ.core :refer [env]]))
+
+(defmacro optional-env-var
+  "For use at compile time. Pulls a var from the environment and
+  returns it."
+  [name]
+  (env name))
```
Comment (alice-dw) [inline src/clj/vit_configurator/env.clj:8]: nit: newline

File: resources/public/staging-preview.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+		  tool.style.width = "320px";
+	      } else {
+		  tool.style.width = "640px";
+	      }
+	  }
+      }
+      window.addEventListener('message', handleEvent);
+
+      window.parent.postMessage("ready");
+      console.log("sent handshake");
```
Comment (alice-dw) [inline resources/public/staging-preview.html:36]: do we need the console logs in this file?

**Review by alice-dw — APPROVED** [review]
👍 lgtm if it works, but tbh not entirely sure on all the parts here so maybe defer to danny for final approval

File: resources/public/staging-preview.html (hunk 2 above)
Comment (dwxander) [reply resources/public/staging-preview.html:36]: I don't think we need them but they were in the existing embed file, so I left them in here

Comment (alice-dw) [reply resources/public/staging-preview.html:36]: ah. makes sense

**Pull request merged**